src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -495,13 +495,13 @@ These settings are available for agent configuration. Some settings depend on yo
       </tbody>
     </table>
 
-An array of `CLASS#METHOD` (for instance methods) and/or `CLASS.METHOD` (for class methods) strings representing Ruby methods for which the agent will try to add a tracer. This doesn't require any code modifications of the classes that define the methods.
+  An array of `CLASS#METHOD` (for instance methods) and/or `CLASS.METHOD` (for class methods) strings representing Ruby methods that the agent can automatically add custom instrumentation to. This doesn't require any modifications of the source code that defines the methods.
```

Comment:
```suggestion
  An array of `CLASS#METHOD` (for instance methods) and/or `CLASS.METHOD` (for class methods) strings represent Ruby methods that the agent can automatically add custom instrumentation to. This doesn't require any modifications of the source code that defines the methods.
```

Reply:
Hi @ally-sassman! Thanks for the update! 

Just so you know, this is an autogenerated file. We're asking folks who edit this file to let the Ruby agent team know so that they can update the content on our end to make sure your changes aren't erased the next time we generate the file. 

I noticed it this time, so I'll update it directly. In the future, could you please let the team know in the `#ruby-agent` Slack channel or open an issue in https://github.com/newrelic/newrelic-ruby-agent?

Reply:
Oh, wait. I'm not sure I understand this tense change. Could you explain? 

If we take out the parens, it reads:
`An array of class#method and/or class.method strings representing Ruby methods...`

Maybe there's another way we can write the sentence so that's a bit clearer?